mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/CassandraMailboxSessionMapperFactory.java

```diff
@@ -74,7 +74,7 @@ public MailboxMapper createMailboxMapper(MailboxSession mailboxSession) {
 
     @Override
     public AttachmentMapper createAttachmentMapper(MailboxSession mailboxSession) {
-        return new NoopAttachmentMapper();
+        return new CassandraAttachmentMapper(session);
```

Comment:
Yes, but we have to keep it as other implementation may need it (used for commit, rollback...).